bundles/org.openhab.persistence.influxdb/README.md

```diff
@@ -65,16 +67,14 @@ You can also add additional tags for structuring your data. For example, you can
 
 The item configuration will look like this:
 
-```
-Group:Number:AVG gTempSensors
-
-Number:Temperature tempLivingRoom (gTempSensors) { influxdb="temperature" [floor="groundfloor"] }
-Number:Temperature tempKitchen (gTempSensors) { influxdb="temperature" [floor="groundfloor"] }
-
+```java
+    Group:Number:AVG gTempSensors
```

Comment:
Can you remove this added unneeded indentation here and below?
```suggestion
Group:Number:AVG gTempSensors
```

Reply:
Done